### PR TITLE
#57 CYC-89 Changed app name and icon logo

### PR DESCRIPTION
**Pull request by bcobo341**

### CODE REVIEW

**Comment by willyyhuang** [comment]
please fix merge conflict

**Comment by willyyhuang** [comment]
also test is not running, debug it please

**Comment by willyyhuang** [comment]
i realized this branch is very old, please pull latest main

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (willyyhuang) [inline resources/views/welcome.blade.php]: I believe all these should be in the layout template instead right? @m-triassi

Comment (m-triassi) [reply resources/views/welcome.blade.php]: Actually i don't think this view even gets used anymore. iirc it was deleted in some other pull request, and this is restoring it for some reason.

**Review by m-triassi — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**